app/src/main/res/values/strings.xml

```diff
@@ -655,4 +655,5 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="label_housenumber">house number</string>
     <string name="label_blocknumber">block number</string>
     <string name="quest_busStopShelter_covered">Whole stop is covered</string>
+    <string name="quest_leave_new_note_photos_are_useful">Adding photo will make the report much more useful.</string>
```

Comment:
Either singular:
```suggestion
    <string name="quest_leave_new_note_photos_are_useful">Adding an photo will make the report much more useful.</string>
```

…or (IMHO better) plural:
```suggestion
    <string name="quest_leave_new_note_photos_are_useful">Adding photos will make the report much more useful.</string>
```

Reply:
Right, but it is "a photo", right?

Reply:
yep, edited

Reply:
Thanks for reminding me that a/an/the exists. I went with singular as single photo in every note would be a massive improvement, requesting multiple ones may be too much.